docs/InstallGuide.md

```diff
@@ -1,129 +1,136 @@
+![vnet_minimum](https://www.dropbox.com/s/dezarv561fg7sdj/vnet_minimum.png?dl=1)
 
-### System Requirements
+On a given physical server (here named as server1) there is one virtual network whose network address is 10.100.0.0/24 and 2 virtual machines joining it.
+The orange circles describe OpenVNet's ruby processes; vna, vnmgr and webapi.
+See architecture for more detail of how the OpenVNet works.
+
+
+## Requirements
 
-+ RHEL 6.4
-+ Kernel 2.6.32-358.6.2.el6.x86_64
 
 ### Network Requirements
 
 + Local Area Network (LAN)
 + Internet connection
 
-Pre-setup
----------
+## Installation
+
+### Install OpenVNet Packages
 
-Download the openvnet.repo file and put it to your /etc/yum.repos.d/ directory.
+Download the openvnet.repo file and put it to your `/etc/yum.repos.d/` directory.
 
     # curl -o /etc/yum.repos.d/openvnet.repo -R https://raw.githubusercontent.com/axsh/openvnet/master/openvnet.repo
 
-Download the openvnet-third-party.repo file and put it in your /etc/yum.repos.d/ directory.
+Download the openvnet-third-party.repo file and put it in your `/etc/yum.repos.d/` directory.
 
     # curl -o /etc/yum.repos.d/openvnet-third-party.repo -R https://raw.githubusercontent.com/axsh/openvnet/master/openvnet-third-party.repo
 
-Install epel-release.
+Each repo has the following packages:
 
-    # rpm -Uvh http://dlc.wakame.axsh.jp.s3-website-us-east-1.amazonaws.com/epel-release
+* openvnet.repo
+  * openvnet (virtual package)
+  * openvnet-common
+  * openvnet-vna
+  * openvnet-vnmgr
+  * openvnet-webapi
 
+* openvnet-third-party.repo
+  * openvnet-ruby
+  * openvswitch
 
-OpenVNet installation
-------------------------
+Install epel-release.
 
-Install OpenVNet.
+    # yum install -y epel-release
+
+
+Install OpenVNet packages.
 
     # yum install -y openvnet
 
-Initial Configuration
----------------------
+`openvnet` is a virtual package. It is equivalent to install `openvnet-common`,
+`openvnet-vna`, `openvnet-vnmgr` and `openvnet-webapi` at once.
 
-### Setup vnmgr
+### Edit Configuration Files
 
-Modify the following section in /etc/openvnet/vnmgr.conf on the vnmgr node.
+Edit the file `/etc/openvnet/vnmgr.conf`
 
     node {
       id "vnmgr"
       addr {
         protocol "tcp"
-        host "192.168.100.2"
+        host "127.0.0.1"
+        public ""
         port 9102
       }
     }
 
-- **host** : The IP address or host name of the ZeroMQ node.
-- **port** : The port number for connecting with ZeroMQ. If the vnmgr process is run on the same node as vna or webapi, specify a different value from vna and webapi.
+Modify the parameters `host` and `public` according to your environment. In order for
+the sample environment in the overview section we leave those parameters as is. The detail of each parameter is following.
 
+- **id** : The identifier of the OpenVNet's process. For example the paramter `id` in `vnmgr.conf` is applied to the vnmgr process. Specify universally unique string data in the world of the OpenVNet.
```

Comment:
I think this line is a little confusing. **id** is used by 0mq right? I would suggest this:

"OpenVNet uses the [0mq](http://zeromq.org) protocol for communication between its processes. This id is used by 0mq to identify each process. Any string here is fine as long as there's no two processes using the same one. It's recommended to just leave the default values."